docs-ref-conceptual/format-output-azure-cli.md

```diff
@@ -195,10 +195,21 @@ KBDemo020
 
 The `tsv` output format is often used when assigning values to variables. This example gets the active subscription ID and stores it into a variable for use in a script.
 
+# [Bash](#tab/bash)
+
 ```azurecli-interactive
+# Bash Script
 subscriptionID=$(az account show --query id --output tsv)
 echo "Using subscription ID $subscriptionID"
 ```
+# [PowerShell](#tab/powershell)
+
+```azurecli
+# PowerShell script
+$subscriptionID = az account show --query id --output tsv
+Write-Host "Using subscription ID $subscriptionID"
```

Comment:
```suggestion
echo "Using subscription ID $subscriptionID"
```

Reply:
@v-abdriaz , `write-host` is correct but it doesn't work in Bash.  `echo` works in both environments.